packages/extension-sdk/src/connect/extension_host_api.ts

```diff
@@ -115,12 +118,20 @@ export class ExtensionHostApiImpl implements ExtensionHostApi {
       }
       case ExtensionNotificationType.VISUALIZATION_DATA: {
         const { payload } = message
-        ;(this.visualizationSDK as VisualizationSDK).updateVisData(payload)
+        this.visualizationSDK.updateVisData(payload)
         if (this.visualizationDataReceivedCallback) {
           this.visualizationDataReceivedCallback(payload)
         }
         return undefined
       }
+      case ExtensionNotificationType.TILE_HOST_CHANGED: {
+        const { payload } = message
+        this.tileSDK.tileHostDataChanged(payload)
+        if (this.tileHostChangedCallback) {
+          this.tileHostChangedCallback(payload)
```

Comment:
Is `tileHostDataChanged` a sub-event of `tileHostChangedCallback`?

Reply:
Good catch. I renamed it and I guess I missed that one. Another change coming

Reply:
Done